<launch>

  <!-- motor node -->
  <include file="$(find xycar_motor)/launch/xycar_motor_a2.launch" />
  <include file="$(find obstacle_detector)/launch/test.launch" />

  <!-- usb_cam -->
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/videoCAM" />
    <param name="autoexposure" value="false"/>
    <param name="exposure" value="60"/> 
    <param name="image_width" value="1280" />
    <param name="image_height" value="720" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>

  <!-- auto drive -->
  <node name="auto_drive" pkg="auto_drive" type="main.py" output="screen"/>

</launch>
